treeherder/model/models.py

```diff
@@ -1251,6 +1251,23 @@ def elastic_search_insert(self):
             return es_line
 
 
+class Group(models.Model):
+    """
+    The test harness Group.  Sometimes this is a Manifest, but not always.
```

Comment:
s/Group/group/. But I don't really know what a "test harness group" is. I don't have a great suggestion for wording here, but I think we can do better.